pkg/subctl/cmd/benchmark.go

```diff
@@ -78,7 +78,11 @@ func checkBenchmarkArguments(args []string, intraCluster bool) error {
 }
```

Comment:
These should probably also be handled, but in a different PR

Reply:
Yes, that’s the plan ;-).